packages/model/src/research-output.ts

```diff
@@ -295,11 +297,17 @@ export type ResearchOutputPostRequest = {
   type: ResearchOutputType;
   usageNotes?: string;
   usedInPublication?: boolean;
-  publishingEntity: ResearchOutputPublishingEntities;
 };
 
 export type ResearchOutputPutRequest = ResearchOutputPostRequest;
 
+type NonEmptyArray<T> = [T, ...T[]];
+
+export type ResearchOutputWorkingGroupPostRequest =
```

Comment:
do we need a separate type for the Post request?

Reply:
I'd say we need one since we have a error message that can be either addressed to teams or working groups. The researchOutputData used there is of type ResearchOutputPostRequest and I'm using this type in the type guard.